pkg/controller/job/job_controller.go

```diff
@@ -1284,6 +1290,18 @@ func (jm *Controller) recordJobFinished(job *batch.Job, finishedCond *batch.JobC
 	return true
 }
 
+// pastActiveDeadline checks if job has ActiveDeadlineSeconds field set and if
+// it is exceeded. If the job is currently suspended, the function will always
+// return false.
+func (jm *Controller) pastActiveDeadline(job *batch.Job) bool {
```

Comment:
nit: Please move this function where it was (below `pastBackoffLimitOnFailure`)

Reply:
/ack